driver/index.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const events = require('../eventPool');
+const pickupHandler = require('./handler');
```

Comment:
If you are having trouble running it, try wrapping the `pickupHandler` in curly bracket